lib/manageiq/providers/ibm_cloud/cloud_tools/vpc_sdk.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require 'ibm_vpc'
+
+dir_name = File.basename(__FILE__).split('.')[0]
+Dir.glob(File.join(dir_name, '*.rb'), :base => File.dirname(__FILE__)) { |req| require_relative req }
+
+module ManageIQ
+  module Providers
+    module IbmCloud
+      module CloudTools
+        module VpcSdk
+        end
+
+        class Vpc < ManageIQ::Providers::IbmCloud::CloudTools::Base::Sdk
```

Comment:
I'm not sure I understand why a module of VpcSdk was created (which matches the file name), but then also Vpc was created (which should be in its own file).

Reply:
or should this be _inside_ VpcSdk?

Reply:
Vpc is the class that is used. VpcSdk is the namespace that organizes the subclasses used by the Vpc class.

Reply:
There should have been a comment on the VpcSdk to be used for documentation.

Reply:
I would recommend those living in separate files named vpc_sdk.rb and vpc.rb, respectively

Reply:
Files split.